generic-client/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "solana-generic-client"
```

Comment:
We already have solana-client, this solana-generic-client makes the other less "generic" which is not true. I think the original module files are mostly for BenchTpsClient, I think we can drop the bench only and name this solana-tps-client.